src/addon/downloader.cpp

```diff
@@ -152,9 +152,9 @@ TransferStatusList::push(TransferStatusListPtr statuses)
 
 // Called when one of the transfers completes.
 void
-TransferStatusList::on_transfer_complete(TransferStatusPtr this_status, bool success)
```

Comment:
Why not `successful`?

Reply:
I later get a "variable shadows local variable error", that's what the last commit fixes.

Reply:
This name seems better, will change.